pages/api/users/models/UserSchema.ts

```diff
@@ -1,11 +1,8 @@
-import { Schema, model } from 'mongoose';
+import { Schema, model, models } from 'mongoose';
 import bcrypt from 'bcryptjs';
-
-enum Gender {
-    MALE = 'male',
-    FEMALE = 'female',
-    OTHER = 'other',
-}
+import isEmail from 'validator/lib/isEmail';
```

Comment:
@JRakin we don't need to use any third party package for validation. We have our own 
https://www.npmjs.com/package/js-string-helper